appservice/src/main/java/nu/yona/server/analysis/rest/AppActivityController.java

```diff
@@ -52,4 +58,28 @@ public void addAppActivity(@RequestHeader(value = PASSWORD_HEADER) Optional<Stri
 			return null;
 		});
 	}
+
+	@RequestMapping(value = "/", method = RequestMethod.GET)
+	@ResponseBody
+	@ResponseStatus(HttpStatus.OK)
+	public HttpEntity<AppActivityDTO[]> getAppActivities(@RequestHeader(value = PASSWORD_HEADER) Optional<String> password,
+			@PathVariable UUID userID)
+	{
+		// This method is just a shorthand to facilitate link building
```

Comment:
I don't think this is necessary or good to do. The intention is to share a link to which the user can POST app activities. Therefore the link can use the `addAppActivity` method and the `getAppActivities` method is not necessary.